tests/test_data_loaders.py

```diff
@@ -125,6 +125,43 @@ def test_load_resource_spec_unmodeled_object_property_missing_additional_propert
     )
 
 
```

Comment:
think you can also just put these schemas in [`tests/data/schema/invalid`](https://github.com/aws-cloudformation/cloudformation-cli/tree/master/tests/data/schema/invalid) without having to maintain additional test code

Reply:
not necessary, bc I will have to adjust a ton of tests. I made just two tests with local schemas so it wont affect whole lot of tests